src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/CreateMakeDecisionDashboardNotificationForRespondentHandler.java

```diff
@@ -50,6 +43,12 @@ protected String getScenario(CaseData caseData) {
             && caseData.getMakeAppVisibleToRespondents() != null) {
             return SCENARIO_AAA6_GENERAL_APPLICATION_JUDGE_UNCLOAK_RESPONDENT.getScenario();
         }
+        if (isWithNoticeOrConsent(caseData)
```

Comment:
Could you plz refactor this to something like 

if (isWithNotceOrConsent(caseData)) {
     if (condition1)  return scenario1;
     else if ( condition 2) return scenario 2;
     
    As for my current ticket, I will be adding 1 more condition for withNoticeOrConsent application.

Reply:
Done.